auth-web/src/views/auth/staff/ContinuationAuthorizationReview.vue

```diff
@@ -138,4 +262,35 @@ export default class ContinuationAuthorizationReview extends Vue {
 
 <style lang="scss" scoped>
 @import '$assets/scss/theme.scss';
+
+// add whitespace to the right of the container
+@media (min-width: 960px) {
+  .view-container {
+    padding-right: 5rem;
+  }
+}
+
+@media (min-width: 1264px) {
+  .view-container {
+    padding-right: 10rem;
+  }
+}
+
+@media (min-width: 1360px) {
```

Comment:
Do we really need this? I think it's already defined in `layout.scss`. But, I'm guessing you need to overwrite it with yours since you want it to look different right Sev?

Reply:
`layout.scss` does not include the above CSS for whitespace to the right of the view container. The above are actually custom values for this page (to match the designs). And yes, keeping them here will make this component more portable. I'd like to leave this alone.